docker/oso-saml-sso/src/root/aws_saml_cert.yml

```diff
@@ -0,0 +1,25 @@
+# autogenerated on Wed Aug 30 09:10:29 EDT 2017
+# from https://signin.aws.amazon.com/static/saml-metadata.xml
+---
+aws_saml_subject: /CN=urn:amazon:webservices/O=Amazon Web Services, Inc./ST=Washington/C=US
+aws_saml_expire: 1535155200
+aws_saml_x509: |
```

Comment:
Has this already been tested, and still formats the cert data in the same way?

Reply:
Yes.